admin/sql/create_tables.sql

```diff
@@ -104,4 +104,11 @@ CREATE TABLE follow_list (
 );
 ALTER TABLE follow_list ADD CONSTRAINT follow_list_name_creator_key UNIQUE (name, creator);
 
+CREATE TABLE recommendation (
```

Comment:
I thought we were gonna create a seperate [schema](https://www.postgresql.org/docs/9.5/ddl-schemas.html) for this.